layers/+lang/scala/packages.el

```diff
@@ -66,7 +66,7 @@
         "Regenerate `.ensime' file and restart the ensime server."
         (interactive)
         (progn
-          (sbt-command "gen-ensime")
+          (sbt-command ";ensimeConfig;ensimeConfigProject")
```

Comment:
It's required, see http://www.scala-sbt.org/0.13/docs/Howto-Running-Commands.html#Provide+multiple+commands+to+run+consecutively